tests/integrations/remotes/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+services:
+  minio:
+    container_name: minio
+    image: minio/minio
```

Comment:
maybe we need a tag to make the CI reliable ?

Reply:
Oh yes definitely. Pinning tags is a good idea